### ISSUE TITLE
Newer Version Doesn't Open On Windows XP x64 Anymore On Both Supermium And Thorium

### ISSUE THREAD

**Issue by MWF2** [IDA-RE-things/Chrome-xp-api-adapter#16]
hello, thanks for making that api, and also after i put newer version of that, it don't open on xp x64 anymore, i did read before i put on both supermium and thorium folder, and it's 32 bit of browser on 64 bit os, after this when i try to open, it don't open anymore
![image](https://github.com/IDA-RE-things/Chrome-xp-api-adapter/assets/74994144/8f496af0-3540-4596-be17-bd41eb4cf410)

**Comment by IDA-RE-things** [comment]
Hello, the XP x64 was not supported officially by me curretly. But it should work fine in WOW3264-mode.
It was disabled. I will enable it in next build, ok. I think today.

**Comment by IDA-RE-things** [comment]
Have updated build 5059. Now should work and on XP x64

**Comment by EDSln** [comment]
Yes, now works on Windows XP x64, hardware acceleration works the same way. 
Now I wish there was a 64 bit version... 😉

**Comment by IDA-RE-things** [comment]
Hello :),
Native XP x64 version will be in the near feature. Just requires some time to rework/refactor the code. It was developed initially only for x86. Stay watch the project
